.gitignore

```diff
@@ -52,5 +52,13 @@ Brewfile.lock.json
 # Node modules
 /node_modules
 
+# Ignore ovpn files
+*.ovpn
```

Comment:
I think not all these folders are getting created since we changed the approach of cert generation for the Client to Site VPN. Maybe none of them is needed anymore. But We can probably test by running an example.

Reply:
Let's tackle this separately as not related to the move to github.com itself.